modules/system/classes/Vite.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace System\Classes;
+
+use Illuminate\Foundation\Vite as LaravelVite;
+use Illuminate\Support\Facades\App;
+use Illuminate\Support\HtmlString;
+use Winter\Storm\Exception\SystemException;
+
+class Vite extends LaravelVite
+{
+    /**
+     * Generate Vite tags for an entrypoint(s).
+     *
+     * @param string|array $entrypoints The list of entry points for Vite
+     * @param string|null $package The package name of the plugin or theme
+     * @return HtmlString
+     *
+     * @throws SystemException
+     */
+    public function __invoke($entrypoints, $package = null)
+    {
+        if (!$package) {
+            throw new SystemException('A package must be passed');
```

Comment:
Should this be an InvalidArgumentException instead? Why do we initialize it to null if we then perform a falsey check on it and reject if it fails?

Reply:
Because it's overloading the parent's behaviour which is to execute on `__invoke`, however in the parent [Vite](https://github.com/laravel/framework/blob/11.x/src/Illuminate/Foundation/Vite.php) the second param is "buildPath" which:

A. We don't need.
B. We replace with `package` to allow us to manage which package is being used.

We can't modify the type hint to not allow null because php no like so have to validate in the method's logic.

I've updated the exception tho :)